### ISSUE TITLE
UI

### ISSUE THREAD

**Issue by xandros-na** [ShawnCramp/intellileague#1]
We should build a mock-up of the app @ShawnCramp

**Comment by ShawnCramp** [comment]
Yeah I was thinking that too when I was in the lab.  If you're in the lab tomorrow we can.  I just wrote down some ideas of what we can do, but I have no idea how to make the app look lol or what we want each controller view to contain

**Comment by ShawnCramp** [comment]
Only view I started was just the login page.  So when the app is opened, it asks for your summoner name, then you move onto the next view after entering

**Comment by xandros-na** [comment]
Ok, we can do a one-time login page with username and password since we're doing builds as well (they can sign-out later if they choose to). Then we can decide what to show after the login page. We can take some design ideas from this [app](https://play.google.com/store/apps/details?id=com.lolsummoners)

**Comment by ShawnCramp** [comment]
Sounds good to me.  Maybe we can have like a register page if you want to do username and password, then we can let them fill in their summoner name so they get information on themselves.  We can always store information locally for app close and open like we did in assignment two.  Then we can make a simple rest database and host it on hopper or something to store username and password information.  Up to you for what you want to do.

**Comment by xandros-na** [comment]
Oh yeah, I forgot about the register page lol. Yeah that sounds good to me.

**Comment by xandros-na** [comment]
We should use a tab bar in place of the buttons in the view after the log in screen. It's more convenient to access the other views that way rather than tapping the back button until you get to the view that has these buttons: Summoner info, champs, builds, items.

**Comment by ShawnCramp** [comment]
Good idea.  I was just messing around with the index screen this morning, but all I did was place some buttons and images to see how it looked, but using a tab bar sounds like a good idea.  I was following a tutorial to get the size navigation working, looks like we have to use this [SWRevealViewController](https://github.com/John-Lluch/SWRevealViewController) to get the best results.  I didn't quite have time to finish the tutorial, so I was going to go back and finish that tomorrow

**Comment by xandros-na** [comment]
Kk, there's also a [native API](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UISplitViewController_class/) from apple and this is the [tutorial](http://nshipster.com/uisplitviewcontroller/) if we hit problems with the SWRevealViewController.

**Comment by ShawnCramp** [comment]
Okay sounds good
